Test.Integration.Net7/DecisionsAPI/DecisionsRequests.cs

```diff
@@ -0,0 +1,61 @@
+using Sift;
+using Test.Integration.Net7.Uitlities;
+using Xunit;
+
+namespace Test.Integration.Net7.DecisionsAPI
+{
+    //Ignore
+    public class DecisionsRequests
+    {
+        private readonly EnvironmentVariable environmentVariable = new();
+
+        [Fact]
+        public void IntegrationTest_GetDecisionStatusRequest()
+        {
+            var sift = new Client(environmentVariable.ApiKey);
```

Comment:
is it possible to read the key somewhere in the class constructor one time, and just use the key in the methods below? Instead of reading it from env for every single method run

Reply:
@binish-exalture pls, take a look